apps/dashboard/app/views/batch_connect/shared/_app_list.html.erb

```diff
@@ -27,5 +26,5 @@
       )
     %>
   <%- end -%>
-  </div>
+  </ul>
```

Comment:
Why change the tag of the header to "a" from "p"? Semantically that does not seem correct.